01_science_technology_and_epistemology/01_science_technology_and_epistemology.jl

```diff
@@ -8,72 +8,186 @@ using InteractiveUtils
+We are makers. We are Hackers. And, as makers that we are, we learn by doing. 
+That is another sign of how different technology is from the sciences. 
+Scientists don't learn science by doing it, but by doing labs and problem sets. 
+Scientists start out doing work that's perfect, in the sense that they're just trying to reproduce work someone else has already done for them. 
+Eventually, they get to the point where they can do original work. Whereas makers, from the start, are doing original work; it's just very bad. 
+So makers start original, and get good, and scientists start good, and get original.
 
-## What is Science?
-
-As a first approach, science is a method that was conceived to help humans discover the principles -invariant laws- that describe the world. And note that we said "method", because this is all it really is: A methodology which ensures that we are as objective and data-driven as possible.
-
-So, science is a method we have built to conclude that hypotheses we propose are not false. This doesn't mean they are true, and this is one of the core concepts of the scientific method. There are no dogmas in science. Scientific propositions, claims, hypotheses or theories can be tested to try to falsify it. If there isn't a way to test their validity, then they aren't scientific propositions, and it is not a matter of science to discuss such statements. Rather, we can debate them in the realms of philosophy and other disciplines. Science happens in the context of a community; it makes no sense to talk about an hypothesis being false by its own, there must be a community that can validate and replicate the evidence. Moreover, the descriptions we do about nature with science are not absolute, but just the best we can say. As the famous physicist Niels Bohr once said, when answering questions about quantum physics: "There is no quantum world. There is only an abstract quantum physical description. It is wrong to think that the task of physics is to find out how nature is. Physics concerns what we can say about Nature."
-
-But nowadays things are a little bit different. The experimentation is not being as rigorous as it used to be and, in some cases, it's totally left behind. The pressure to produce academic "knowledge" and our multiple biases and beliefs (almost religious in some cases) are interfering with the progress of science.
+So then, what role does Science play in all this and why is it useful for technology? As we already said, it is a tool that technology has to formalize all that chaotic based knowledge discovery. 
```

Comment:
> So then,

> So,

Reply:
I still see capitalization on certain words that don't need it, be sure to check the whole chapter and correct them.

Reply:
> science is a tool that technology uses to formalize the chaotic process of knowledge discovery. 

Try to polish sentences a bit more

Reply:
Changed by:

"So, what role does science play in all this and why is it useful for technology? As we already said, it is a tool that technology has to formalize all that chaotic based knowledge discovery. 
Formalization is very useful to teach and communicate some base knowledge and to build a solid foundation in order to keep expanding it. 
But it is also the way we found to draw out subjectivities regarding the understanding of our world."